src/components/cc-invoice-table/cc-invoice-table.js

```diff
@@ -46,20 +45,16 @@ export class CcInvoiceTable extends withResizeObserver(LitElement) {
     /** @type {Invoice[]|null} Sets the list of invoices. */
     this.invoices = null;
 
-    /** @type {number|null} */
-    this._width = null;
-  }
-
-  onResize ({ width }) {
-    this._width = width;
+    new ResizeController(this);
```

Comment:
issue: let's keep a reference to the controller so we can get its `width` property

```js
this._resizeController = new ResizeController(this);
```